components/utils/ResourceInfoCard.tsx

```diff
@@ -4,7 +4,7 @@ import { Edit, Trash } from 'tabler-icons-react';
 export interface ResourceInfoCardProps {
   resourceType: string;
   label: string;
-  date: string | React.ReactElement;
+  date: string[];
```

Comment:
Rather than making this an Array of string where we know it will only have two values and the first and second both correspond with a specific value, I would make use of this interface. I would just make date a string and then add another attribute to this interface for `datePath` or something like that.